TC003_Pomis1AllAreas.py::TC001_Pomis1_Report::test_Hakimpur_Area
--------------------------------------------------------------------
Last Page: https://timf.imikrof.com/PKSF_POMIS_1_REPORT
  Browser: chrome (103.0.5060.114)
   Driver: chromedriver (103.0.5060.53)
Timestamp: 1657103413  (Unix Timestamp)
     Date: Wednesday, 6 July 2022
     Time: 4:30:13 PM  (Bangladesh Standard Time, UTC+06:00)
--------------------------------------------------------------------
Traceback: File "C:\Users\TMSS-ICT\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 591, in run
    self._callTestMethod(testMethod)
  File "C:\Users\TMSS-ICT\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 549, in _callTestMethod
    method()
  File "E:\python-workplace\tmss-ict\iMikrof\TestsCases\Pomis1Report\TC003_Pomis1AllAreas.py", line 653, in test_Hakimpur_Area
    self.assert_equal(exp_samity_total, act_samity_total)
  File "C:\Users\TMSS-ICT\AppData\Local\Programs\Python\Python310\lib\site-packages\seleniumbase\fixtures\base_case.py", line 5908, in assert_equal
    self.assertEqual(first, second, msg=msg)
  File "C:\Users\TMSS-ICT\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Users\TMSS-ICT\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 1226, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Users\TMSS-ICT\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
Exception: '445' != '307'
- 445
+ 307
